openproblems/tasks/dimensional_reduction/metrics/pearsons_chi_squared.py

```diff
@@ -0,0 +1,15 @@
+
+from ....tools.decorators import metric
+
+
+@metric(metric_name="Pearson's chi squared test", maximize=True)
+def chi_squared(adata, method):
+
+    """
+    Perform Pearson's chi squared test to estimate the goodness of fit of dimensional reduction.
+    """
+
+    from scipy.stats import chisquare
+
+    adata.obsp["chi_squared"] = chisquare(adata.obsm["dimensional_reduction_method"], f_exp=adata.X.astype(array))
```

Comment:
Metric should return a float, at the moment it looks like you're just modifying in place?

Reply:
Yeah this is semi-updated for now - I need to verify that summation of this method into a single float is accepted. will follow up shortly.

Reply:
Other than this chi-squared metric, this is good to go for now. I might delete it for now so we can close this branch/PR and then update with a new metric in a new PR

Reply:
@scottgigante ok so I ended up removing it for now ([commit](https://github.com/singlecellopenproblems/SingleCellOpenProblems/pull/92/commits/8fc866a7514e90272004f8f7b1781d8243165b49)). I think everything is good to go now - can we close this branch and add to the website? lmk if there's anything I am missing!